lstchain/calib/camera/calib.py

```diff
@@ -48,10 +46,14 @@ def lst_calibration(event, telescope_id):
     dc2pe = event.mc.tel[telescope_id].dc_to_pe  # numgains * numpixels
     signals *= dc2pe
 
-    event.dl1.tel[telescope_id].image = signals
+    threshold = 4094
```

Comment:
we had this threshold defined somewhere else (I think it was for the previous gain_selection...), but we could maybe define it globally for the LST

Reply:
See PR #142 for global definition ;-)

Reply:
Great! :D
The checks are failing in that PR because of the problem on this one, right?

Reply:
yes!